autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -147,9 +143,15 @@ def parse_course(course_data: List,
     credit_hours = course_data['creditHourLow']
 
     # Parse the instructor, then send the returned Instructor model to parse_section
-    instructor_model = parse_instructor(course_data, instructors_set)
+    instructor_model = parse_instructor(course_data)
     section_data = parse_section(course_data, instructor_model)
 
+    if not instructor_model is None and instructor_model.id not in instructors_set:
+        instructors_set.add(instructor_model.id)
```

Comment:
```suggestion
    if instructor_model is not None:
        instructors_set.add(instructor_model)
```
Hashing a model just hashes its primary key so it's fine to pass in the instructor directly, and there's no reason to check if something is in a set before adding it: if it's already a member, it'll just do nothing.
Also `is not` is an operator

Reply:
Will change the `is not` operator, but we want to set `instructor_model` to None whenever the `instructor_model` is already in the `instructor_set`, so I'm leaving that check. Otherwise it would set `instructor_model` to None whenever it is None, which doesn't make much sense.